packages/react-native/lib/trustx-biometric-plugin.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  IDV_EVENTS,
+  IDVProvider,
+} from '@docknetwork/wallet-sdk-core/src/biometric-provider';
+import {IWallet} from '@docknetwork/wallet-sdk-core/src/types';
+import {EventEmitter} from 'stream';
+import * as Keychain from 'react-native-keychain';
+import axios from 'axios';
+
+export type TrustXIDVConfig = {
+  walletApiUrl: string;
+};
+
+async function performBiometricCheck() {
+  await Keychain.getGenericPassword({
+    accessControl: Keychain.ACCESS_CONTROL.BIOMETRY_ANY,
+  });
+}
+
+export function createTrustXIDVProviderFactory({
+  wallet,
+  eventEmitter,
+  configs,
+}: {
+  wallet: IWallet;
+  eventEmitter: EventEmitter;
+  configs: TrustXIDVConfig;
+}): IDVProvider {
+  return {
+    async enroll(walletDID, proofRequest) {
+      
+      // TODO: get start URL from TrustX API
```

Comment:
when can we come back for this TODO?

Reply:
need to remove this comment, it was already done